cli/generator-op-microservice/generators/microservice/templates/rest.service.ts

```diff
@@ -134,20 +131,19 @@ export class <%= serviceClassName %> {
    */
   public routes() {
 
-    // configure CORS
-    const corsOptions: cors.CorsOptions = {
-      allowedHeaders: ['Origin', 'X-Requested-With', 'Content-Type', 'Accept', 'X-Access-Token'],
-      credentials: true,
-      methods: 'GET,HEAD,OPTIONS,PUT,PATCH,POST,DELETE',
-      origin: '*',
-      preflightContinue: false
-    };
-
-    this.app.use(cors(corsOptions));
-    // create API routes
-    this.app.use('/', <%= apiName %>Router);  }
+  // configure CORS
+  const corsOptions: cors.CorsOptions = {
+    allowedHeaders: [ 'Origin', 'X-Requested-With', 'Content-Type', 'Accept', 'X-Access-Token' ],
+    credentials: true,
+    methods: 'GET,HEAD,OPTIONS,PUT,PATCH,POST,DELETE',
+    origin: '*',
+    preflightContinue: false
+  };
+
+  this.app.use( cors( corsOptions ) );
+  // create API routes
+  this.app.use( '/', <%= apiName %> Router );
+}
 }
```

Comment:
Shouldn't one of these be on a different tab index?

Reply:
updated